azure/requests.yaml

```diff
@@ -61,23 +63,6 @@ releases:
         - name: cluster-operator
           version: ">= 0.23.20"
           issue: https://github.com/giantswarm/roadmap/issues/187
-
-    - name: ">= 12.0.0 < 13.0.0"
```

Comment:
Are these cleanup for non-existing releases? In the future might make sense to do these in separate PR :)

Reply:
yeah right, thanks